spring-cloud-stream-binder-jms-common/src/main/java/org/springframework/cloud/stream/binder/jms/config/JmsBinderConfigurationProperties.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.cloud.stream.binder.jms.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Donovan Muller
+ */
+@ConfigurationProperties("spring.cloud.stream.binder.jms")
+public class JmsBinderConfigurationProperties {
+
+    private String deadLetterQueueName = "dlq";
```

Comment:
Should we perhaps use a more specific name for the DLQ ? "Spring.Cloud.Stream.DLQ" or something like that?